sass/partials/_blog.scss

```diff
@@ -67,6 +67,28 @@ article {
   .entry-content ol {
     margin-left: .9em;
   }
+  .entry-content > table {
+    width: 100%;
+    margin-bottom: 1.5em;
+
+    thead {
+      border-bottom: 2px solid lighten($quality-color, 45);
+      th {
+        font-weight: bold;
+        padding-bottom: 5px;
+        padding: 5px;
```

Comment:
I changed padding size to em. Border size should be keep in px, since its a value that won't change based on font size. Furthermore other border values are on px.